src/TestFramework/Coverage/JUnitTestFileDataProvider.php

```diff
@@ -36,38 +36,55 @@
 namespace Infection\TestFramework\Coverage;
 
 use DOMDocument;
+use DOMElement;
+use DOMNodeList;
 use DOMXPath;
+use function file_exists;
+use function Safe\sprintf;
+use Webmozart\Assert\Assert;
 
 /**
  * @internal
  */
 final class JUnitTestFileDataProvider implements TestFileDataProvider
 {
-    private $jUnitFilePath;
+    private $jUnitPath;
 
     /**
      * @var DOMXPath|null
      */
     private $xPath;
 
-    public function __construct(string $jUnitFilePath)
```

Comment:
I would like to preserve this name as it immediately clear that the path is to file, not directory of files (as for XML coverage)